src/service/viewport/viewport-binding-ios-embed-sd.js

```diff
@@ -404,6 +404,24 @@ export class ViewportBindingIosEmbedShadowRoot_ {
         + this.getBorderTop();
   }
 
+  /** @override */
+  contentHeightChanged() {
+    if (isExperimentOn(this.win, 'scroll-height-bounce')) {
+      // Refresh the overscroll (`-webkit-overflow-scrolling: touch`) to avoid
+      // iOS rendering bugs. See #8798 for details.
+      this.vsync_.mutate(() => {
```

Comment:
we only need to do this once and the very first time `scrollHeight > height` subsequent height changes will be fine. That would guarantee user can not be scrolling since we are going from non-scrollable to scrollable which should resolve @jridgewell's concern below.

so maybe a variable to keep track of this has been done (or use `once` from  `function.js` and a check on `if(this.scroller_.scrollHeight > contentHeight)`

Reply:
For scrolling - yes, doing it once is sufficient. Unfortunately, for rendering - it's not.

Reply:
RE: scrolling impact - I responded on the other thread.